packages/components/src/components/DetailsHeader/DetailsHeader.jsx

```diff
@@ -126,11 +133,16 @@ export default function DetailsHeader({
   if (type === 'taskRun') {
     ({ reason: reasonToUse, status: statusToUse } = getStatus(taskRun));
     statusLabel =
-      reasonToUse ||
-      intl.formatMessage({
-        id: 'dashboard.taskRun.status.pending',
-        defaultMessage: 'Pending'
-      });
+      reason === 'tkn-dashboard:skipped'
```

Comment:
Would it be worth storing this string as a constant? As it's used across quite a few files

Reply:
yep, done